### PR TITLE
Object recognizer with model rcnn inception

### PR DESCRIPTION
**Pull request by ferAguilar02**

### CODE REVIEW

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+public class RCNN implements ModelRecognizer {
+    RcnnDetection object;
+    @Override
+    public List<MatchInfo> matching(Path pathFolder, String searchCriteria, int occurrencyPercentage) {
+        List<MatchInfo> matches = new ArrayList<>();
+        try {
+            File file = pathFolder.toFile();
+            String[] listImg = file.list();
+            if (listImg == null || listImg.length == 0) {
+                System.out.println("There are no elements inside the current folder");
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java]: Please remove this `System.out.println()`

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizertypes/rcnn/RcnnDetection.java [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+import ai.djl.repository.zoo.ZooModel;
+import ai.djl.training.util.ProgressBar;
+import ai.djl.translate.Batchifier;
+import ai.djl.translate.TranslateException;
+import ai.djl.translate.Translator;
+import ai.djl.translate.TranslatorContext;
+import ai.djl.util.JsonUtils;
+import com.google.gson.annotations.SerializedName;
+
+import org.jalau.at18.searchobject.common.exception.FaceDetectionException;
```
Comment (rodri-98) [inline searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizertypes/rcnn/RcnnDetection.java]: Please remove this unused import

**Review by rodri-98 — CHANGES_REQUESTED** [review]
Please remove all the files that were generated at the moment to run the models, the folder uploads has to be empty, in the same way the folder resources/Images has to be empty

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package org.jalau.at18.searchobject.model.objectrecognizer.recognizer;
+
+import org.jalau.at18.searchobject.model.objectrecognizer.recognizertypes.MatchInfo;
+import org.jalau.at18.searchobject.model.objectrecognizer.recognizertypes.rcnn.RcnnDetection;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
```
Comment (DaniMB95) [inline searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java]: add documentation to the model

Comment (DaniMB95) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java]: @ferAguilar02 erase the upload images

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java (hunk 1 above)
Comment (ferAguilar02) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java]: I have removed the System.out and replaced it with a logger

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizertypes/rcnn/RcnnDetection.java (hunk 2 above)
Comment (ferAguilar02) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizertypes/rcnn/RcnnDetection.java]: Import was removed

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java (hunk 3 above)
Comment (ferAguilar02) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java]: I have added documentation and removed all images

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java (hunk 1 above)
Comment (ferAguilar02) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/objectRecognizer/recognizer/RCNN.java]: All images in uploads were erased

**Review by rodri-98 — APPROVED** [review]
Nice job!

**Review by saraialvarezrosas — APPROVED** [review]

**Review by DaniMB95 — APPROVED** [review]

**Pull request merged**